src/modules/load_mon/load_mon.cpp

```diff
@@ -68,10 +69,10 @@ namespace load_mon
 
 extern "C" __EXPORT int load_mon_main(int argc, char *argv[]);
 
-// Run it at 1 Hz.
-const unsigned LOAD_MON_INTERVAL_US = 1000000;
+// Run it at 10 Hz.
+const unsigned LOAD_MON_INTERVAL_US = 100000;
```

Comment:
Good for bench-testing but I would not increase it in general.

We can add spike-detection instrumentation to the `top` command. Then we can also do something that is a bit more expensive.